dir.props

```diff
@@ -45,19 +45,45 @@
     <Platform Condition="'$(Platform)'==''">AnyCPU</Platform>
```

Comment:
Multiplying the configurations comes with a maintenance cost. So if there is no value, we should leave it as simple as possible, IMO.